common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -379,4 +379,13 @@ trait FeatureSwitches {
     exposeClientSide = true
   )
 
+  //Owener Dotcom participation
+  val MobileLabPresidentialPrimaries = Switch(
+    SwitchGroup.Feature,
+    "mobile-lab-presidential-primaries",
+    "When ON, we present a promo banner to android, US edition users inviting them to subscribe to experimental primaries alerts",
+    safeState = Off,
+    sellByDate = new LocalDate(2016, 6, 7),
```

Comment:
I would run this until Wednesday as with the time difference people should be able to sign up until Tuesday (as the test is running on Tuesday)